### PR TITLE
Release/0.0.1

### PR DESCRIPTION
**Pull request by ZigZagT**
setup auto publishing to npm in this pr

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -16,3 +16,14 @@ script:
   - yarn lint
   - yarn test
   - yarn coveralls <./coverage/lcov.info
+
+jobs:
+  # include:
+  allow_failures:
```
Comment (SBoudrias) [inline .travis.yml:22]: That was for testing right?

Comment (ZigZagT) [reply .travis.yml:22]: no, it doesn't work since I don't have permission to publish. so using `allow_failures` to make CI happy for now

Comment (SBoudrias) [reply .travis.yml:22]: Ah I understand :+1:

File: commit-msg [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
```
Comment (SBoudrias) [inline commit-msg]: Let's remove this one, then we should be good to move forward.

Comment (ZigZagT) [reply commit-msg]: removed

File: packages/ab-testing/package.json [hunk 3]
```diff
@@ -1,18 +1,18 @@
 {
```
Comment (SBoudrias) [inline packages/ab-testing/package.json:1]: We need to add the license in our `package.json` files too

Comment (ZigZagT) [reply packages/ab-testing/package.json:1]: added

**Review by SBoudrias — APPROVED** [review]

**Pull request merged**